xds/src/main/proto/centraldogma/xds/group/v1/xds_group.proto

```
@@ -46,15 +46,21 @@ service XdsGroupService {
 message CreateGroupRequest {
   string group_id = 1 [(google.api.field_behavior) = REQUIRED];
   Group group = 2 [(google.api.field_behavior) = REQUIRED];
+
+  // The commit summary for this change. If not specified, a default message is used.
+  string summary = 3 [(google.api.field_behavior) = OPTIONAL];
```

Buf (1.71.0)

[error] cannot find `google.api.field_behavior` in this scope

(COMPILE)